config/commit-template

```diff
@@ -0,0 +1,5 @@
+The title
```

Comment:
Yeah, I don't care what. My point is, with this template, I'm literally deleting every line and fillign in my own anyways, so the template doesn't provide much value. The template should just be prompts imo.

Maybe:

```

Changes:

Fixes:

/cc @getsentry/team
```

I dunno, I just want this to actually be useful.